recipes-bsp/u-boot/files/ae350-ax45mp/0002-Prevent-fw_dynamic-relocation.patch

```diff
@@ -0,0 +1,29 @@
+From 8e9620fc6b594520cd049f50ab5af438e051ecab Mon Sep 17 00:00:00 2001
+From: Yu Chien Peter Lin <peterlin@andestech.com>
+Date: Fri, 5 Aug 2022 15:53:59 +0800
+Subject: [PATCH] Prevent fw_dynamic relocation
+
+This patch makes u-boot to load fw_dynamic to link address, it is
+intended to skip firmware relocation.
+
+Signed-off-by: Yu Chien Peter Lin <peterlin@andestech.com>
```

Comment:
Same here

Reply:
This patch has been removed, use config fragment to set the OpenSBI load address.